.github/workflows/test.yml

```diff
@@ -90,3 +104,4 @@ jobs:
       working-directory: ${{runner.workspace}}/build
       shell: bash
       run: ctest -C ${{ matrix.type }} --exclude-regex '.*RNDSANDWICHTEST'
+      if: ${{ matrix.os != 'macos-12' && matrix.target-platform != 'iphoneos' && matrix.target-platform != 'iphonesimulator' }}
```

Comment:
Shouldn't this be rather:
${{ matrix.os != 'macos-12' || (matrix.target-platform == 'host-os' }}

And maybe add a comment on top that tests can't be executed on iOS builds to clarify the complex condition a bit

Reply:
Done.